src/main/java/it/pagopa/ecommerce/commons/client/NpgClient.java

```diff
@@ -620,6 +625,7 @@ private NpgResponseException exceptionToNpgResponseException(
         return new NpgResponseException(
                 "Error while invoke method for build order",
                 errors,
+                Optional.empty(),
```

Comment:
Here we can go also in case of `WebClientResponseException`. In that case this status code will not be populated. I would rather declare the optional status code before the `if (err instanceof WebClientResponseException e) {` line and assign it to an empty optional and override it's value into the if body. This way status code is valued for all paths

Reply:
You're right, fixing it

Reply:
Fixed in e946dda